src/algebra/order/group/bounds.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Yury Kudryashov
+-/
+import order.bounds.basic
+import algebra.order.group.basic
+
+/-!
+# Least upper bound and the greatest lower bound in linear ordered additive commutative groups
```

Comment:
Those lemmas belong in `algebra.order.group.basic`

Reply:
Can you please move these again after #17381 lands.

Reply:
That would re-introduce `data.set.intervals.basic` into `algebra.order.group.basic`.